src/jobs/webhookConsumer.ts

```
@@ -46,25 +46,30 @@ export async function startWebhookConsumer(): Promise<void> {
         //  Failed delivery
-        if (retries >= MAX_RETRIES) {
+        if (result.terminal || retries >= MAX_RETRIES) {
           logger.error("Webhook failed permanently", {
             webhookId,
             retries,
           });
 
-          // send to DLQ
-          ch.nack(msg, false, false);
+          // send to DLQ explicitly
+          ch.sendToQueue(QUEUES.WEBHOOKS_DLQ, msg.content, { persistent: true });
```

ESLint

[error] Replace `·persistent:·true` with `⏎············persistent:·true,⏎·········`

(prettier/prettier)